src/content/general/management-clusters/index.md

```diff
@@ -25,21 +25,36 @@ As well as operators, our management clusters run supporting components such as
+There are a number of components that are not a part of a workload cluster release, such as our monitoring stack, or web UI. These components are deployed exclusively to the management cluster, and do not affect workload clusters.
+
+These components are deployed via app collections, which are a set of App Custom Resources, which are ultimately deployed via the App Platform. This collection contains the latest tagged version of each component, and is managed by our deployment component [draughtsman](https://github.com/giantswarm/draughtsman/).
 
-The components are deployed as App custom resources using our App Platform. Changes are tested in a number of internal management clusters before being deployed to all management clusters.
+These management cluster-only components are tested on our internal management clusters before releases are cut.
+
+We plan on replacing draughtsman in Q2 2021, with a new component that more natively synchronises app collections to a management cluster, mostly to improve the scalability of the system.
 
 ## Configuration changes
 
-From time to time, we also need to update the configuration for management clusters. These changes are also tested on our internal management clusters before being applied.
+Periodically, we need to update the configuration for management clusters.
+
+This configuration is currently held in a ConfigMap and a Secret stored on the Management Cluster. When updates are required, these resources are updated manually using our current tooling, from our central configuration repository.
+
+These configuration files are watched by the App Platform, and any necessary applications are reconfigured on changes.
+
+Changes to configuration are tested on our internal management clusters before being applied.
+
+This configuration system is being updated in Q1 2021, with a system based on the new [config-controller](https://github.com/giantswarm/config-controller/). This will remove the need for manual updates of configuration, remove the shared ConfigMap and Secret, and allow for versioning of configuration. Overall, aiming to further increase the safety, reliability, and scalability of configuration changes.
 
 ## Infrastructure upgrades
 
-The infrastructure including the management cluster nodes and supporting resources like bastion hosts are managed by our SRE team using Terraform.
+The management cluster infrastructure, such as the Kubernetes cluster itself, or other supporting resources such as bastion hosts, are managed using Terraform.
+
+Updates to our Terraform playbooks are rolled out by CI as needed, with changes to our Terraform playbooks being tested on internal management clusters before being deployed.
```

Comment:
I thought Terraform was modules and Ansible was playbooks? It's a good name but just not used to seeing it with TF.

Reply:
yeah you're fucking right, i'm confusing the two